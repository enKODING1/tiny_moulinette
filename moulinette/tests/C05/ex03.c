#include <stdio.h>

#define ARR_LEN 9

int ft_recursive_power(int nb, int power);

int main()
{
    int arr[ARR_LEN] = {0,
                        -2147483648,
                        -5,
                        -1,
                        1,
                        2,
                        3,
                        4,
                        5};

    for (int i = 0; i < ARR_LEN; i++)
    {
        printf("%d\n", ft_recursive_power(arr[i], i));
    }
    printf("%d\n", ft_recursive_power(arr[2], -2));
    return 0;
}