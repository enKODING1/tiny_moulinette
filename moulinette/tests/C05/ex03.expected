1
-2147483648
25
-1
1
32
729
16384
390625
0
